test/test_autograd.py

```diff
@@ -6,9 +6,9 @@
 import unittest
 from copy import deepcopy
 from collections import OrderedDict
+from torch.autograd import gradcheck
 
-from common import make_jacobian, TestCase, iter_tensors, \
```

Comment:
Do we still need to keep `make_jacobian` and `get_numerical_jacobian` in `common.py`? They are used as well in `common_nn.py`, but can't we use instead the ones from `autograd` that you just added?